.scala-steward.conf

```diff
@@ -13,6 +13,8 @@ updates.ignore = [
 ]
 
 updates.pin = [
+  // Stay on ScalaTest 3.1 as long as  Akka TestKit does
```

Comment:
Are `//` allowed here for comments?

Reply:
I'd believe this is Typesafe Config, so yes. For consistence I changed to `#`.

Reply:
TIL HOCON supports both `#` and `//` :laughing: